tools/importer-rest-api-specs/components/schema/builder.go

```diff
@@ -378,11 +378,10 @@ func (b Builder) findCreateUpdateReadPayloads(input resourcemanager.TerraformRes
 	out.createModelName = *createOperation.RequestObject.ReferenceName
 	out.createPayload = createModel
 	createPropsModelName, createPropsModel := out.getPropertiesModelWithinModel(out.createPayload, b.models)
-	if createPropsModelName == nil || createPropsModel == nil {
-		return nil, fmt.Errorf("couldn't find `Properties` model for Create Payload")
+	if createPropsModelName != nil || createPropsModel != nil {
+		out.createPropertiesPayload = *createPropsModel
+		out.createPropertiesModelName = *createPropsModelName
```

Comment:
As both `ModelName` and `Payload` for the `Properties` object are becoming optional rather than required - we should [update the `operationPayloads` struct to make these pointers to signify that](https://github.com/hashicorp/pandora/blob/19980c54093303a2d500de7e9b84c15c4b82a27e/tools/importer-rest-api-specs/components/schema/internal.go#L7-L22) - which'd make this more obvious that these are optional in other parts of the codebase

Reply:
As discussed this change causes unwanted effects in the existing generated resources, details in #3588